src/OddJobs/Cli.hs

```diff
@@ -5,12 +5,12 @@
 module OddJobs.Cli where
 
 import Options.Applicative as Opts
+import Data.Coerce (coerce)
 import Data.Text
 import OddJobs.Job (startJobRunner, Config(..), LogLevel(..), LogEvent(..))
 import OddJobs.Types (UIConfig(..), Seconds(..), delaySeconds)
--- import System.Daemonize (DaemonOptions(..), daemonize)
-import qualified System.Posix.Daemon as Daemon
-import System.FilePath (FilePath)
+import qualified System.Posix.Daemonize as Daemon
```

Comment:
I think it's better to keep the alias the same as the last segment of the module name, or use something much shorter. Having this slight difference could be confusing.